lib/auth/saml.go

```diff
@@ -336,14 +388,22 @@ func (a *Server) ValidateSAMLResponse(samlResponse string) (*SAMLAuthResponse, e
 		}
 		return nil, trace.Wrap(err)
 	}
+
 	event.Status.Success = true
 	event.User = re.auth.Username
-	event.Code = events.UserSSOLoginCode
+	if testFlow {
+		event.Code = events.UserSSOTestFlowLoginCode
+	} else {
+		event.Code = events.UserSSOLoginCode
+	}
```

Comment:
https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else

Reply:
TBH I think this rule makes it **harder** to read code, not easier, but yeah, not gonna die on that hill.